### PR TITLE
feat(levm): run all posible transaction of each testcase

### PR DESCRIPTION
**Pull request by maximopalopoli**
**Motivation**

Currently the EF tests handles the first posible transaction, the one with the first data, gas limit and value. We want to handle all the posible combinations between those fields.

**Description**

Currently, the transaccions are handled and prints a report of each test group

### CODE REVIEW

File: cmd/ef_tests/levm/report.rs [hunk 1]
```diff
@@ -2,52 +2,102 @@
...
+
+#[derive(Debug, Default, Clone)]
+pub struct EFTestReport {
     passed: u64,
     failed: u64,
-    total: u64,
-    passed_tests: Vec<String>,
-    failed_tests: Vec<(String, String)>,
+    run: u64,
+    // passed_tests: Vec<String>,
```
Comment (ilitteri) [inline cmd/ef_tests/levm/report.rs:22]: Try not to propose commented code when merging to `main`.
```suggestion
```

File: cmd/ef_tests/levm/report.rs [hunk 2]
```diff
@@ -2,52 +2,102 @@
...
 
 impl EFTestsReport {
     pub fn register_pass(&mut self, test_name: &str) {
-        self.passed += 1;
         self.passed_tests.push(test_name.to_string());
-        self.total += 1;
+
+        let report = self.test_reports.entry(test_name.to_string()).or_default();
+        report.passed += 1;
+        //report.passed_tests.push(tx_indexes);
```
Comment (ilitteri) [inline cmd/ef_tests/levm/report.rs:32]: Ditto.

**Review by ilitteri — APPROVED** [review]
LGTM. Left a couple of minor comments. We can merge after resolving them.

File: cmd/ef_tests/levm/report.rs (hunk 1 above)
Comment (maximopalopoli) [reply cmd/ef_tests/levm/report.rs]: Done both in [Remove commented code](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1175/commits/5c7d971ca53da3183a80427500dea84522f0baeb)

**Pull request merged**